aioxmpp/xso/model.py

```diff
@@ -2571,15 +2571,16 @@ class XSOParser:
         # let Message be a XSO class, like in the XSO example
         result = None
         def catch_result(value):
-            nonlocal result
+            global result
```

Comment:
Ah, this is tricky. Often you'd have this code in a function, where `nonlocal` is the correct choice. But on the global scope that doesn't work :/.

I don't care much either way, so let's keep the new one.